src/bin/real-rat.rs

```diff
@@ -118,7 +166,7 @@ define_language! {
         "-R" = RSub([Id; 2]),
         "*R" = RMul([Id; 2]),
         "/R" = RDiv([Id; 2]),
-        Var(egg::Symbol),
+        Var(Variable),
```

Comment:
why this change?

Reply:
Parsing issue. It was actually causing derive to crash.

Reply:
weird, ok!